std/node/buffer.ts

```diff
@@ -1,601 +1,7 @@
-  }
-
-  writeDoubleBE(value: number, offset = 0): number {
-    new DataView(this.buffer, this.byteOffset, this.byteLength).setFloat64(
-      offset,
-      value,
-    );
-    return offset + 8;
-  }
-  writeDoubleLE(value: number, offset = 0): number {
-    new DataView(this.buffer, this.byteOffset, this.byteLength).setFloat64(
-      offset,
-      value,
-      true,
-    );
-    return offset + 8;
-  }
-
-  writeFloatBE(value: number, offset = 0): number {
-    new DataView(this.buffer, this.byteOffset, this.byteLength).setFloat32(
-      offset,
-      value,
-    );
-    return offset + 4;
-  }
-  writeFloatLE(value: number, offset = 0): number {
-    new DataView(this.buffer, this.byteOffset, this.byteLength).setFloat32(
-      offset,
-      value,
-      true,
-    );
-    return offset + 4;
-  }
-
-  writeInt8(value: number, offset = 0): number {
-    new DataView(this.buffer, this.byteOffset, this.byteLength).setInt8(
-      offset,
-      value,
-    );
-    return offset + 1;
-  }
-
-  writeInt16BE(value: number, offset = 0): number {
-    new DataView(this.buffer, this.byteOffset, this.byteLength).setInt16(
-      offset,
-      value,
-    );
-    return offset + 2;
-  }
-  writeInt16LE(value: number, offset = 0): number {
-    new DataView(this.buffer, this.byteOffset, this.byteLength).setInt16(
-      offset,
-      value,
-      true,
-    );
-    return offset + 2;
-  }
-
-  writeInt32BE(value: number, offset = 0): number {
-    new DataView(this.buffer, this.byteOffset, this.byteLength).setUint32(
-      offset,
-      value,
-    );
-    return offset + 4;
-  }
-  writeInt32LE(value: number, offset = 0): number {
-    new DataView(this.buffer, this.byteOffset, this.byteLength).setInt32(
-      offset,
-      value,
-      true,
-    );
-    return offset + 4;
-  }
-
-  writeUInt8(value: number, offset = 0): number {
-    new DataView(this.buffer, this.byteOffset, this.byteLength).setUint8(
-      offset,
-      value,
-    );
-    return offset + 1;
-  }
-
-  writeUInt16BE(value: number, offset = 0): number {
-    new DataView(this.buffer, this.byteOffset, this.byteLength).setUint16(
-      offset,
-      value,
-    );
-    return offset + 2;
-  }
-  writeUInt16LE(value: number, offset = 0): number {
-    new DataView(this.buffer, this.byteOffset, this.byteLength).setUint16(
-      offset,
-      value,
-      true,
-    );
-    return offset + 2;
-  }
-
-  writeUInt32BE(value: number, offset = 0): number {
-    new DataView(this.buffer, this.byteOffset, this.byteLength).setUint32(
-      offset,
-      value,
-    );
-    return offset + 4;
-  }
-  writeUInt32LE(value: number, offset = 0): number {
-    new DataView(this.buffer, this.byteOffset, this.byteLength).setUint32(
-      offset,
-      value,
-      true,
-    );
-    return offset + 4;
-  }
-}
-
-export default { Buffer };
+export {
+  default,
+  Buffer,
+  kMaxLength,
+  kStringMaxLength,
+  SlowBuffer,
+} from "https://esm.sh/safe-buffer@5.2.0";
```

Comment:
Relying on a third party domain and service here is definitively bit of a no-no from my perspective since std is supposed to be secure and vetted. This extends the trust to an outside source.

Any particular reason we can't just copy this over?